src/main.cc

```diff
@@ -93,13 +93,17 @@ auto jsonschema_main(const std::string &program, const std::string &command,
     return sourcemeta::jsonschema::cli::encode(arguments);
   } else if (command == "decode") {
     return sourcemeta::jsonschema::cli::decode(arguments);
-  } else {
+  } else if (command == "help") {
     std::cout << "JSON Schema CLI - v"
               << sourcemeta::jsonschema::cli::PROJECT_VERSION << "\n";
     std::cout << "Usage: " << std::filesystem::path{program}.filename().string()
               << " <command> [arguments...]\n";
     std::cout << USAGE_DETAILS;
     return EXIT_SUCCESS;
+  } else {
+    std::cerr << "error: Unknown command '" << command << "'.\n";
```

Comment:
```suggestion
    std::cerr << "error: Unknown command '" << command << "'\n";
```

For consistency with all other error messages